sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+_Font Màgica_ gently sounds within a technics and phenomenology of material transformation. Neither a work _of_ traditionally-constructed originality nor one _about_ appropriation, it is a lensing apparatus, a technical-sonic system that is _complicit with_ Chopin and solar flares and Izabella's visuals and my signal processing algorithms. Though slightly different each time due to random signal changes, _Font Màgica_ is not an "open" work so much as an obsessed one. It is obsessed with its own material closing in on itself via signal processing. Chopin's _Raindrop_ appears everywhere, replicated at all levels as both a structuring force as well as a microsonic substrate. _Font Màgica_ aims for a form of systemic closure and inner self-similarity; a closure that paradoxically "realizes openness in its radical sense: not as openness toward the possibility of contingencies from the outside, but as a 'being opened' by the contingent materials that form the work" (Negarestani 3).
+
+Algorithms paint an artwork into a corner. They limit possibilities rather than opening a work up to them. A "rigorous and twisted mode of closure," algorithms allow a work to close in on itself and its materials (2). Too often algorithmic art is couched in a rhetoric of power and control that demands a distinctly perspectival position, like the abstract, theoretical, architectural view from above in De Certeau's reading of the city from the top floor a skyscraper (92-3). For me, algorithms are not about the control afforded by such an over-arching perspective on a system, nor are they simply a Cageian opening up of the work to the contingencies of a constructed "nature" distinct from the technics used to register it. I use algorithms in my work specifically in order to have my view of the aesthetic terrain partially obscured, to find localized energies that emerge not from the abstract adherence to rules, nor openness to chance, but from a kind of obsessive practice that allows a creative system to start feeding on itself, to "uncover itself as the field of experimentation of its contingent materials as a conspiracy plotted by anonymous materials" (Negarestani 4).
+
+### Chris Welsby's Colour Separation
+
+_Font Màgica_'s soundtrack, though perhaps obliquely, invokes for me the work of Chris Welsby, whose films form "cinema systems" that bind together, both as material and method, the technology of the camera and the energies of the natural environment. In Welsby's films, the wind, waves, sun, rain, and clouds all are allowed to imprint themselves onto the chemical, mechanical, optical, and durational surfaces of his cinema. Nature and technology are not opposites, but form an ecology. "Mind, technology and nature are not experienced as separate things divided along Cartesian lines but as interconnected parts of one larger, dynamic system" (Welsby, "Technology, Nature, and Software" 102).
```

Comment:
Comment from David: "I usually flag (for myself) any paragraph that ends with a quotation. Properly speaking a quotation should be in the middle of a paragraph… it doesn’t matter, of course, but when they’re at the end it often signals something that should be explicated or expanded. (I think that’s the case of your Welsby paragraph…and actually, i also think you should draw out the resonances of the connection further so that they become meaningful. i.e: what do we learn about each of the terms being related through relating them? There are millions of possible analogous pieces, but you chose this one)"